api/src/test/scala/org/globalnames/resolver/api/QueryParserSpec.scala

```diff
@@ -0,0 +1,118 @@
+package org.globalnames.resolver.api
+
+import org.scalatest.FunSpec
+import org.globalnames.resolver.api.QueryParser.{Modifier, SearchPart}
```

Comment:
Let's use in the file beginning:

```scala
package org.globalnames
package resolver
package api
```

then you can just type `import QueryParser.{Modifier, SearchPart}`